docs/source/challenge_scenarios/extracting/shared_operations.rst

```diff
@@ -0,0 +1,40 @@
+===========================================
+Sharing Operations Across Multiple Extracts
+===========================================
+
+Scenario
+========
+
+You have ten data files, and you want to do the same set of extraction
+operations in all of them.
+
+Solution with Module Imports
+============================
+
+The ingest library comes with a method for turning any file anywhere into an
+importable module as long as you know where the file is.
+
+So in this example solution we would make a shared_operations.py file
+containing whatever set of operations we want to share, and we'll put it
+somewhere outside of our designated extract_configs directory like this::
+
+    my_study/
+    ├── ingest_package_config.py
+    ├── extract_configs/
+    │   ├── my_extract_config.py
+    │   └── ...
+    ├── my_shared_operations.py     <- contains operations I want to share
+    └── transform_module.py
+
+Then in the extract configuration files, we can import the shared
+operations like this:
+
+.. code-block:: Python
```

Comment:
Can I do this:

```python
from copy import deepcopy

operations = deepcopy(operations)

operations.append(
    constant_map(
        out_col=CONCEPT.GENOMIC_FILE.REFERENCE_GENOME,
        m=constants.SEQUENCING.REFERENCE_GENOME.HS37D5
    )
)
```

Reply:
I'm pretty sure that you don't need to copy there

Reply:
Tried it, you do need the deepcopy. I think you either need to use the approach you've documented or use the deepcopy. You can't just import from `kf_ingest_packages.packages.my_package.extract_configs.shared_config import operations` if you have two or more extract configs importing the shared operations modifying them.

Reply:
> Tried it, you do need the deepcopy

Then I think your example code snippet is missing something.

Reply:
√ I added this too